coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1114,6 +1115,50 @@ func (s *server) notifyWorkspaceBuildFailed(ctx context.Context, workspace datab
 	}
 }
 
+func (s *server) notifyWorkspaceManualBuildFailed(ctx context.Context, workspace database.Workspace, build database.WorkspaceBuild) {
+	templateAdmins, err := s.Database.GetUsers(ctx, database.GetUsersParams{
+		RbacRole: []string{codersdk.RoleTemplateAdmin},
+	})
+	if err != nil {
+		s.Logger.Error(ctx, "unable to fetch template admins", slog.Error(err))
```

Comment:
These log messages aren't really going to help operators figure out what went wrong since there isn't much content.
Can you wrap this logger maybe? Or possibly better yet: move out all of this preparatory work to a helper function and log that the preconditions for this notification failed with reason "X"?